src/styles/main.scss

```
@@ -0,0 +1,53 @@
+.clock {
+  position: relative;
+  width: 500px;
+  height: 500px;
+  margin: auto;
+
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.clock__sec {
```

у меня были похожие замечания, пользуйся аперандом в таких случаях
```suggestion
&__sec {
```